### DISCUSSION TITLE
Offline i18n ?

### DISCUSSION THREAD

**Discussion by KaKi87** [tolgee/tolgee-platform#1048] [Q&A] [answered]
Hello,

Can an app using the Tolgee JS integration load i18n data offline, and more specifically, be locally loaded from the production bundle, like it would be when using `vue-i18n` for example ?

Thanks

**Comment by JanCizmar** [comment, marked as answer]
Hi there! It's described in Preparing for Production section of all integration docs. For [Vue JS](https://tolgee.io/docs/web/using_with_vue/preparing_for_production), it's here.

**Reply by KaKi87** [reply]
Thanks !

I suppose a CLI tool will be mostly useful for generating those JSON files, actually.

**Comment by JanCizmar** [comment]
Thats right!
